.eslintignore

```diff
@@ -1,2 +1,2 @@
 /lib-es5
-/test
\ No newline at end of file
+node_modules
```

Comment:
Why did we ignore test to begin with?

Reply:
Yes, make sense, test shall be also linted